certified-connectors/Bitvore/apiDefinition.swagger.json

```diff
@@ -0,0 +1,773 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "The Entity API provides access to the information aboutcorporate entities. \n\n Companies can be searched for based on some criteria or directly using a Bitvore ID which uniquely identifies an entity. When using the Bitvore ID the entire company profile will be returned for that single company. In addition, a search can be performed using different profile properties such as name, ticker, industry, etc. The API will return all companies that match the search criteria.",
```

Comment:
The new lines in the description unfortunately won't work, so they can be removed